Gruntfile.js

```diff
@@ -92,6 +92,7 @@ module.exports = function(grunt) {
   // on each recompile
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
+      .replace(/\r\n/g, "\n")
```

Comment:
sed:privateDefinitions works on the current files in the directory, though.  How does fixing up .gitattributes affect the functionality of sed:privateDefinitions?